public/js/common/export/export.js

```diff
@@ -37,8 +37,10 @@ export var ExportModal = React.createClass({
     },
 
     getGraphImage: function() {
+        var JsonLocalStorageObj = JSON.stringify(localStorage);
```

Comment:
Yeah I totally agree with you and actually I thought about when writing this code. But I didn't see how to filter out necessary courses for current graph... Or maybe I can delete some LS like 'bool1' (they are really minority of LS though)?